numpy/fft/pocketfft/pocketfft.c

```diff
@@ -9,18 +9,12 @@
  *  Copyright (C) 2004-2018 Max-Planck-Society
  *  \author Martin Reinecke
  */
-#define NPY_NO_DEPRECATED_API NPY_API_VERSION
```

Comment:
The changes in this file are just stripping out the previous numpy call code, making the file as close as possible to the original (e.g., removing the added `static`).